### DISCUSSION TITLE
GitHub Education Benefits

### DISCUSSION THREAD

**Discussion by serdestgun** [community/community#135082] [GitHub Education]
### Hi everyone,

It has been one week since my application has been approved. Yet, I still can't see the benefits of GitHub Education. Is there anyone who can check?

**Comment by ahmedembeddedxx** [comment]
First, check if your request was approved to verify your academic status. You can do this by visiting this [link](https://education.github.com/pack/offers). I applied a week ago, and my status was approved 5 days later. 

If your status is approved, keep in mind that GitHub is currently experiencing a high volume of requests. To address any issues, you can submit a support ticket using this [link](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket). With some luck, your ticket will be resolved promptly.

**Comment by davevad93** [comment]
Hi @serdestgun ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @serdestgun was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
